_visual/typography.md

```diff
@@ -61,7 +61,7 @@ order: 01
         <div class="usa-grid-full">
           <div class="usa-width-two-thirds">
             <p>A simple serif and sans serif combination designed to communicate warmth and credibility. Strong Merriweather heading weights offer clear information hierarchy and when paired with Source Sans Pro’s easy-to-read body text, creates a clean and professional feel.</p>
-            <p>This is pairing is included in the Web Design Standards framework.</p>
+            <p>This pairing is included in the Web Design Standards framework.</p>
```

Comment:
checking with the #writing-lab to see if Web Design Standards should be capitalized or not. Otherwise, yes.